### ISSUE TITLE
Request Filter information - targets is not a string[]

### ISSUE THREAD

**Issue by louis-lau** [nodemailer/wildduck#144]
According to the docs targets should be a string[]. e.g
```json
"targets": [
  "johndoe@example.com",
  "smtp://mx.example.com:25",
  "https://example.com"
]
```
Instead an object[] gets returned:
```json
"targets": [
  {
    "id": "5d56e7b21fbd9004ecd99ded",
    "type": "mail",
    "value": "johndoe@example.com"
  },
  {
    "id": "5d56e7b21fbd9004ecd99dee",
    "type": "relay",
    "value": "smtp://mx.example.com:25"
  },
  {
    "id": "5d56e7b21fbd9004ecd99def",
    "type": "http",
    "value": "https://example.com"
  }
]
```

Seems like this is just taken directly from MongoDB.
Which is wrong, the docs or the behavior?

EDIT: I would prefer a string[], just for consistency with the other methods.

**Comment by andris9** [comment]
Docs are correct, it should return the same value that was inserted. The problem is that there's a separate client implementation being worked on so I have to check which version they use, maybe it's easier to change the docs

**Comment by andris9** [comment]
It is going be as stated in the docs (list of strings)
Fixed with 9dc18e8a7ebb4b409271b049d9513ddb5d54fb33
